homework/aboutme.js

```diff
@@ -0,0 +1,121 @@
+// - Create a JavaScript file, and link it up
+// - Using JavaScript:
+
+//   -* Change the font-family of the page to `"Arial, sans-serif"`
+//   -* Replace each of the `span` tags (nickname, favorite, hometown) with your own information (e.g. put your nickname in the nickname span)
+//   -* Give each `li` the class of `"listitem"`
+//   -* Change each `li`'s text color to `"rebeccapurple"`
+//   -* Create a new `img` element and set its `src` attribute to a picture of you.
+//     Put that new `img` element in the div with the class of `"profilePicture"`
+
+document.querySelector('body').style.fontFamily='Arial, sans-serif';
+
+const span = document.querySelectorAll('span');
+span[0].innerHTML='Mees';
+span[1].innerHTML='Turtle';
+span[2].innerHTML='Wonder Land';
+
+const list = document.querySelectorAll('li');
+
+list.forEach(e=>{
+  e.setAttribute('class', 'listitem')
+})
+
+const listClass = document.querySelectorAll('.listitem')
+console.log(listClass)
+listClass.forEach(e=>{
+  e.style.color='rebeccapurple'

```

Comment:
You can move this line of code and put it in the previous forEach loop 👍 
```suggestion
  e.style.color='rebeccapurple'
```